app/controllers/vendors_controller.rb

```diff
@@ -1,2 +1,96 @@
 class VendorsController < ApplicationController
+  def index
+    # if request.fullpath == "/vendors"
+    #   @sign_in = "Vendor"
+    # else
+    # id = params[:market_id]
+    # @market = Market.find(id)
+    @vendors = Vendor.all
+    @@sign_in = "Vendor"
+    @sign_in  = "Vendor"
```

Comment:
I'm wondering why you need to have two separate variables that are storing the same type of information